images-list

```diff
@@ -1,6 +1,19 @@
-# This list should be sorted alphabetically. Do not change existing entries. Comments lines may be started with # or //.
+# This list should be sorted lexographically. Do not change existing entries. Comments lines may be started with # or //.
 # This list was reset in order to prevent breakage when moving to multiarch mirroring. See the following PR for more information:
 # https://github.com/rancher/image-mirror/pull/27
 #
 # LIST FORMAT: <SOURCE> <DESTINATION> <TAG>
+coredns/coredns rancher/coredns-coredns 1.7.0
+coredns/coredns rancher/coredns-coredns 1.7.1
```

Comment:
We could only do the version that will end up being live, but this is probably a good test case to check.